universal-login-relayer/test/relayer/services/messages/MessageQueueStore.ts

```diff
@@ -1,53 +1,66 @@
 import {expect} from 'chai';
 import {utils} from 'ethers';
 import {getKnex} from '../../../../lib/utils/knexUtils';
-import TransactionQueueStore from '../../../../lib/services/messages/MessageQueueStore';
+import MessageQueueStore from '../../../../lib/services/messages/MessageQueueStore';
 import {SignedMessage} from '@universal-login/commons';
 import {getTestSignedMessage} from '../../../config/message';
 
-describe('INT: Transaction Queue Store', async () => {
-  let transactionQueueStore: TransactionQueueStore;
+describe('INT: Message Queue Store', async () => {
+  let messageQueueStore: MessageQueueStore;
   let signedMessage: SignedMessage;
 
   before(async () => {
     signedMessage = await getTestSignedMessage();
   });
 
   beforeEach(async () => {
-    transactionQueueStore = new TransactionQueueStore(getKnex());
+    messageQueueStore = new MessageQueueStore(getKnex());
   });
 
   it('construction: queue is empty', async () =>  {
-    const nextTransaction = await transactionQueueStore.getNext();
+    const nextTransaction = await messageQueueStore.getNext();
     expect(nextTransaction).to.be.undefined;
   });
 
-  it('addTransaction', async () =>  {
-    const [idFirst] = await transactionQueueStore.add(signedMessage);
+  it('add message', async () =>  {
+    const [idFirst] = await messageQueueStore.add(signedMessage);
 
     expect(idFirst).to.be.a('number');
     expect(idFirst).to.be.at.least(1);
   });
 
-  it('transaction round trip', async () => {
-    const [idFirst] = await transactionQueueStore.add(signedMessage);
+  it('get message', async () => {
+    const [id] = await messageQueueStore.add(signedMessage);
+    const messageEntity = await messageQueueStore.get(id);
+    expect(messageEntity.message.gasLimit.toString()).to.deep.eq(signedMessage.gasLimit);
+    expect(messageEntity.message.gasPrice.toString()).to.deep.eq(signedMessage.gasPrice);
+    expect(messageEntity.message.value.toString()).to.deep.eq(signedMessage.value);
+    expect(messageEntity.message.gasToken).to.deep.eq(signedMessage.gasToken);
+    expect(messageEntity.message.data).to.deep.eq(signedMessage.data);
+    expect(messageEntity.message.from).to.deep.eq(signedMessage.from);
+    expect(messageEntity.message.to).to.deep.eq(signedMessage.to);
+    expect(messageEntity.message.signature).to.deep.eq(signedMessage.signature);
```

Comment:
`expect(messageEntity.message).to.deep.eq(signedMessage);` or `expect(messageEntity.message).to.deep.include(signedMessage);`